test/helpers.lua

```diff
@@ -225,21 +225,7 @@ local function check_cores(app, force)
     local esigns = ('='):rep(len / 2)
     out:write(('\n%s Core file %s %s\n'):format(esigns, core, esigns))
     out:flush()
-    local pipe = io.popen(
-        db_cmd:gsub('%$_NVIM_TEST_APP', app):gsub('%$_NVIM_TEST_CORE', core)
-        .. ' 2>&1', 'r')
-    if pipe then
-      local bt = pipe:read('*a')
-      if bt then
-        out:write(bt)
-        out:write('\n')
-      else
-        out:write('Failed to read from the pipe\n')
-      end
-    else
-      out:write('Failed to create pipe\n')
-    end
-    out:flush()
+    os.execute(db_cmd:gsub('%$_NVIM_TEST_APP', app):gsub('%$_NVIM_TEST_CORE', core) .. ' 2>&1')
```

Comment:
Looks like now the backtrace is not written to stdout? Or does `os.execute()` do that implicitly?

Reply:
It implicitly does nothing, which means the subprocess inherits stdio.

Reply:
You are missing `out:write('\n')`, AFAIR it was there for a reason, though do not remember whether I just wanted an empty line or the above command sometimes did not produce trailing newline.

And code looks like I was planning to write output to some configurable location instead of simply dumping to stdout always, do not remember what for I needed that exactly.

Reply:
added `\n`. We can revert and debug the pipe code once there is an actual need for it.